### PR TITLE
[4.0] Improve a Language String

### PR DESCRIPTION
**Pull request by astridx**
Improve Language string as suggested in https://github.com/joomla/joomla-cms/issues/23775

### CODE REVIEW

**Comment by brianteeman** [comment]
As this is a new string I would change the KEY as well as the Value

**Comment by astridx** [comment]
@brianteeman Done. Thank you, you are right.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on d9d341f5427bd00368c20dd02e2f57f561831d2c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23855">issues.joomla.org/tracker/joomla-cms/23855</a>.</sub>

File: administrator/language/en-GB/en-GB.com_templates.ini [hunk 1]
```diff
@@ -179,7 +179,7 @@ COM_TEMPLATES_NEW_FILE_TYPE="File Type"
 COM_TEMPLATES_NO_TEMPLATE_SELECTED="No template selected."
 COM_TEMPLATES_OPTION_NONE=":: None ::"
 COM_TEMPLATES_OPTION_SELECT_MENU_ITEM="- Select Menu Item -"
-COM_TEMPLATES_OVERRIDE_ACTION="Update Action"
+COM_TEMPLATES_OVERRIDE_SOURCE="Update Source"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_templates.ini]: Please sort key in alpha order.

Comment (astridx) [reply administrator/language/en-GB/en-GB.com_templates.ini]: @Quy Done. Thanks

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d2dad1a17bfd2fb9d3466d03f84d60f66161e897<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23855">issues.joomla.org/tracker/joomla-cms/23855</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23855">issues.joomla.org/tracker/joomla-cms/23855</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**